lib/Doctrine/ORM/Tools/Pagination/LimitSubqueryOutputWalker.php

```diff
@@ -39,17 +49,17 @@ class LimitSubqueryOutputWalker extends SqlWalker
     /**
      * @var \Doctrine\DBAL\Platforms\AbstractPlatform
      */
-    private $platform;
+    private $_platform;
```

Comment:
@stof Please see http://www.doctrine-project.org/jira/browse/DDC-3688 - this is the issue I was talking about... Any ideas?